pkg/controllers/managementuser/rbac/roletemplates/register.go

```diff
@@ -0,0 +1,13 @@
+package roletemplates
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/pkg/types/config"
+)
+
+func Register(ctx context.Context, workload *config.UserContext) {
+	management := workload.Management.WithAgent("rbac-role-templates")
+
+	management.Management.RoleTemplates("").AddLifecycle(ctx, "cluster-roletemplate-handler", newRoleTemplateLifecycle(workload))
```

Comment:
This is registering a new controller for RoleTemplates in the norman informers, but we could use a wrangler controller instead.
In fact, there are already other uses for this kind:
https://github.com/rancher/rancher/blob/bb98b236ec62b9c0538bd760d81f2e689a6732db/pkg/controllers/management/authprovisioningv2/authprovisioningv2.go#L98
So if we used both, we would be duplicating the caches.